template_bemobil_import.m

```
@@ -0,0 +1,220 @@
+generalInfo.dataset_description.Acknowledgements    = 'acknowledgement text';
+generalInfo.dataset_description.Funding             = {"funding source 1", "funding source 2"};
+generalInfo.dataset_description.ReferencesAndLinks  = {"reference", "link to article"};
+generalInfo.dataset_description.DatasetDOI          = 'DOI of your data set';
+
+% general information shared across modality specific json files 
+generalInfo.InstitutionName                         = 'name of your institute';
+generalInfo.InstitutionalDepartmentName             = 'name of your department';
+generalInfo.InstitutionAddress                      = 'address of your institute';
+generalInfo.TaskDescription                         = 'text describing your task';
+ 
+
+% information about the eeg recording system 
+% will be saved in BIDS-folder/sub-XX/[ses-XX]/eeg/*_eeg.json and *_coordsystem.json
+% see "https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/03-electroencephalography.html#:~:text=MAY%20be%20specified.-,Sidecar%20JSON%20(*_eeg.json),-Generic%20fields%20MUST"
+% and "https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/03-electroencephalography.html#:~:text=after%20the%20recording.-,Coordinate%20System%20JSON,-(*_coordsystem.json"
+%--------------------------------------------------------------------------
+%--------------------------------------------------------------------------
+eegInfo     = [];
+eegInfo.coordsystem.EEGCoordinateSystem     = 'enter the name of your coordinate system'; % only needed when you share eloc
+eegInfo.coordsystem.EEGCoordinateUnits      = 'enter the unit of your coordinate system'; % only needed when you share eloc
+eegInfo.coordsystem.EEGCoordinateSystemDescription = 'enter description of your coordinate system'; % only needed when you share eloc
+eegInfo.eeg.SamplingFrequency               = 1000; % nominal sampling frequency  
```

if the this nominal sampling frequency is not the same as in the XDF stream, I would raise an error